backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxTransformerApi.scala

```diff
@@ -35,7 +34,10 @@ class VeloxTransformerApi extends ITransformerApi with Logging {
    * such as Cast(ArrayType).
    */
   def doValidate(blacklist: Map[String, String], expr: Expression): Boolean = {
-    val value = blacklist.get(expr.prettyName.toLowerCase())
+    val key = if (expr.prettyName.toLowerCase().equals("alias")) {
```

Comment:
Could you add some notes here to explain why `alias` should be specially handled?

Reply:
Done